modules/atomic-widgets/prop-types/border-radius-prop-type.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropTypes;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Base\Object_Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Border_Radius_Prop_Type extends Object_Prop_Type {
```

Comment:
add test in the base class: get data for save

Reply:
Done